### PR TITLE
Radio sound

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR

Now talking into radio and hearing radio messages produces a sound
Now only for headsets
Reference: [#85115](https://github.com/tgstation/tgstation/pull/85115)

## Why / Balance

Don't miss radio messages, greater immersion

## Media

https://github.com/user-attachments/assets/9c4901bc-32af-4d25-acc5-d1e0805f2f1a

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Hearing and talking into the radio now produces a sound

### CODE REVIEW

**Comment by Djungelskog2** [comment]
I'm sorry but I think the current sounds are fine, it sounds super disruptive and hearing it constantly would be like nails against a chalkboard.

**Comment by UbaserB** [comment]
The sound's a little jarring, but I absolutely adore the idea.

File: Content.Server/Radio/EntitySystems/HeadsetSystem.cs [hunk 1]
```diff
@@ -52,6 +54,9 @@ private void OnSpeak(EntityUid uid, WearingHeadsetComponent component, EntitySpo
             && TryComp(component.Headset, out EncryptionKeyHolderComponent? keys)
             && keys.Channels.Contains(args.Channel.ID))
         {
+            if (TryComp<HeadsetComponent>(component.Headset, out var headset))
+                _audio.PlayGlobal(headset.MessageSendSound, uid);
+
```
Comment (metalgearsloth) [inline Content.Server/Radio/EntitySystems/HeadsetSystem.cs:59]: Does this add a global override or just the property?

**Comment by Djungelskog2** [comment]
Additionally I thing I noticed about 13 radio is how it's physically slower to type into and is less active for that, ss14 radio is far more intuitive and faster (for better or for worse), the SFX shouldn't carry the same weight

**Comment by bruhmogus** [comment]
maybe we should use a more staticy version of the speech sound instead? the current noises should be saved for handhelds maybe

**Comment by Unbelievable-Salmon** [comment]
This is extremely useful from an accessibility standpoint and would make my life ten times better with my vision severely limited as it is, so I ask this is greatly considered.

**Comment by themias** [comment]
I like the idea! Perhaps there could be an option under Audio settings to toggle it for those who might not enjoy it.

**Comment by Hrosts** [comment]
Having it togglable per-channel would be nice

**Comment by forgotmyotheraccount** [comment]
Maybe radios could have a menu which lets you toggle which channels produce sound? Not a bad feature, but it'd be unbearable to get pinged for every common message you receive on a high pop server.

**Comment by lizelive** [comment]
1. use per dept sound so you can tell what you are hearing. An easy standard you could use is https://en.wikipedia.org/wiki/DTMF
2. have per channel mute (including common)

**Review by UbaserB — CHANGES_REQUESTED** [review]
After internal discussion, we've decided that there should be a toggle per channel and the sound itself needs to be changed so that it's less jarring.

**Comment by UbaserB** [comment]
You coming back to this?

**Comment by GroggleG** [comment]
Played with this on TG, it was quite literally the most obnoxious thing that was merged in recent years (ignoring the wallening). in anywhere above 20 people it was really fucking annoying.

**Comment by superjj18** [comment]
> Played with this on TG, it was quite literally the most obnoxious thing that was merged in recent years (ignoring the wallening). in anywhere above 20 people it was really fucking annoying.

Can you not simply turn it off in the settings?

**Comment by GroggleG** [comment]
given the fact that most people did I think that should be the default :feelsgood: 

> > Played with this on TG, it was quite literally the most obnoxious thing that was merged in recent years (ignoring the wallening). in anywhere above 20 people it was really fucking annoying.
> 
> Can you not simply turn it off in the settings?

**Comment by Snorkcom** [comment]
nice spam, make the sound mute then

**Comment by superjj18** [comment]
> given the fact that most people did I think that should be the default :feelsgood: 
> 
> 
> 
> > > Played with this on TG, it was quite literally the most obnoxious thing that was merged in recent years (ignoring the wallening). in anywhere above 20 people it was really fucking annoying.
> 
> > 
> 
> > Can you not simply turn it off in the settings?
> 
> 
> 
> 

Not to be that guy but what analytics are you referring to in order to come to the conclusion that "most people did"

**Comment by UbaserB** [comment]
Please ask if you want this PR to be reopened.

**Pull request closed without merging**